src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,378 @@
+be used by a payer to complete a payment. However, before directing a payer to their wallet, the recipient
+MUST append all of the query parameters defined in the [following section](#payid-easy-checkout-url-query-parameters).
+
+Once a PayID Easy Checkout URL is assembled, PayID Easy Checkout is considered to be complete.
+
+### PayID Easy Checkout URL Query Parameters
+This specification defines several query parameter names and corresponding datatypes which MUST be added to the
+PayID Easy Checkout URL before redirecting a payer to their wallet. The PayID Easy Checkout URL SHOULD be parsed 
+by the wallet in order to retrieve any values set by the recipient. It is RECOMMENDED that wallets use these 
+values to pre-populate a payment transaction.
+    
+| Name           | Type             | Description                                                          |
+|----------------|------------------|----------------------------------------------------------------------|
+| amount         | integer          | The amount that should be sent by the payer to the recipient         |
+| receiverPayId  | string           | The [PAYID-URI][] of the receiver                           |
+| assetCode      | string           | The currency code that denominates the amount as defined in [PAYID-PROTOCOL][] |
+| assetScale     | short            | Defines how many units make up one regular unit of the assetCode     |
+| paymentNetwork | string           | The payment network, as defined in [PAYID-PROTOCOL][], that the sender should use to send a payment. |
+| nextUrl        | HTTP Url string  | A URL that the sender's wallet can navigate a sender to after the sender completes a payment  |
+|----------------|------------------|----------------------------------------------------------------------|
+    
+When adding values into a URI 'query' part as defined by
+[RFC3986][], values with characters outside the character set allowed by query parameters in [RFC3986][]
+MUST be percent or otherwise encoded.
+
+Protocols MAY define additional query parameter names and syntax rules, but MUST NOT
+change the meaning of the variables specified in this document.
+
+For example:
+
+    Input:    alice$wallet.com
+              amount=10
+              receiverPayId=pay$merchant.com
+              assetCode=XRP
+              assetScale=6
+              network=XRPL
+              nextUrl=https://merchant.com/thankyou
+    PayID Easy Checkout URL: https://wallet.com/checkout
+    Output:   https://wallet.com/checkout?amount=100000&receiverPayId=payid%2Apay%24merchant.com&assetCode=XRP&assetScale=6&paymentNetwork=XRPL&nextUrl=https://merchant.com/thankyou
+
+# PayID Easy Checkout JRDs
+This section defines the PayID Easy Checkout Link, which conforms to section 4.4 of the
+[Webfinger RFC][RFC7033].  In order for a PayID Discovery Server to enable PayID Easy Checkout, a PayID Discovery query to the server
```

Comment:
```suggestion
[Webfinger][RFC7033].
```

This section should only define the link-relation for the JRD. The `In order for a PayID Discovery Server to enable PayID Easy Checkout, a PayID Discovery query to the server...` is already defined above, and should be struck.